conslogging/conslogging.go

```diff
@@ -195,6 +196,9 @@ func (cl ConsoleLogger) PrintPhaseHeader(phase string, disabled bool, special st
 	defer cl.mu.Unlock()
 	msg := phase
 	c := cl.color(phaseColor)
+	if cl.shouldGroupPhaseOutput() {
+		msg = fmt.Sprintf("::group::%s", msg)
+	}
```

Comment:
To print uncolored output, IIRC, you could try to use `cl.errW.Write([]byte("::group::")` before the line `c.Fprintf(cl.errW, " %s", msg)`.

Reply:
I didn't think of this. Might work. I've updated the PR.

Reply:
You should do the same for endgroup